docs/node/hello-world.md

```diff
@@ -1,7 +1,7 @@
 # Hello world
 
 In this tutorial we'll go over the different components that make up a Temporal project.
-To set it up see the [getting started](./getting-started) instuctions.
+All of the code in this page is included in our package initializer, set it up using the [getting started](./getting-started) instructions.
```

Comment:
```suggestion
All of the code in this page is included in our package initializer, set it up using the [getting started](/docs/node/getting-started) instructions.
```

Prefer to use this format for relative links as it makes it easier to search and replace if needed.